src/extensions/default/AutoUpdate/main.js

```diff
@@ -279,7 +311,8 @@ define(function (require, exports, module) {
 
         updateDomain.exec('initNode', {
             messageIds: MessageIds,
-            updateDir: updateDir
+            updateDir: updateDir,
+            requester: domainID
```

Comment:
Looking at the command name it seems `initNode` is for initialising the node domain. Why do we need `domainID` in that case?